### DISCUSSION TITLE
Welcome back!

### DISCUSSION THREAD

**Discussion by dhakelila** [Vizzuality/mangrove-atlas#335] [General]
Hey @lhilarides and @klongleywood, 

We have started moving! This week we were reviewing the data and where we stand and next week we'll start working with the new datasets thinking about the new widgets. 

Our idea is to arrive at the Friday meeting with some ideas about how those widgets could look like at least for some of them. 

Please, if you had the time to discuss with your team and you got ideas that could help us, please, share so we can use them next week! 

Looking forward to chatting with you next week!

**Comment by lyubovpencheva** [comment]
This is an old one.
